cli.js

```diff
@@ -62,21 +68,37 @@ const uploadColor = string => (data.isDone ? chalk.green(string) : chalk.cyan(st
 
 const downloadColor = string => ((data.isDone || data.uploadSpeed) ? chalk.green(string) : chalk.cyan(string));
 
+const latencyColor = string => (data.isLatencyDone ? chalk.white(string) : chalk.cyan(string));
+const bufferbloatColor = string => (data.isBufferbloatDone ? chalk.white(string) : chalk.cyan(string));
+
 const speedText = () =>
 	cli.flags.upload ?
 		`${downloadColor(downloadSpeed())} ${chalk.dim('/')} ${uploadColor(uploadSpeed())}` :
 		downloadColor(downloadSpeed());
 
-const speed = () => speedText() + lineBreak(2);
+const latencyText = () => `Latency:  ${latencyColor(data.latency + data.latencyUnit)} ${chalk.dim('(unloaded)')}  ${bufferbloatColor(data.bufferbloat + data.bufferbloatUnit)} ${chalk.dim('(loaded)')}`;
```

Comment:
@sindresorhus Is it problematic that we use `chalk.dim` even when stdout is not a TTY?

Reply:
No, Chalk handles that for us.